### PR TITLE
Replace ncp, mkdirp, and rimraf with fs-extra for io.js support

### PR DESCRIPTION
**Pull request by corbinu**
Fixes issue #275 by replacing ncp, mkdirp, and rimraf with fs-extra. Have tested iojs 1.0.4 on OS X. I will add iojs testing to the travis.yml as soon as travis supports it per https://github.com/travis-ci/travis-ci/issues/3108

### CODE REVIEW

File: install.js [hunk 1]
```diff
@@ -351,7 +348,7 @@ function copyIntoPlace(extractedPath, targetPath) {
   .then(function () {
     // Cleanup extracted directory after it's been copied
     console.log('Removing', extractedPath)
-    return kew.nfcall(rimraf, extractedPath).fail(function (e) {
+    return kew.nfcall(fs.remove, extractedPath).fail(function (e) {
```
Comment (nicks) [inline install.js]: i think this is now obsolete? fp.move is equivalent to ncp + rimraf

**Comment by nicks** [comment]
thanks, can you do a quick cla? 
https://github.com/Medium/opensource#note-to-external-contributors

File: install.js (hunk 1 above)
Comment (corbinu) [inline install.js]: Will check

**Comment by corbinu** [comment]
Will do

**Comment by corbinu** [comment]
Signed and removed the unnecessary fs.remove code

**Comment by nicks** [comment]
lgtm!

**Pull request merged**